src/main/java/ua/softserveinc/tc/controller/user/BookingTimeController.java

```diff
@@ -59,7 +54,8 @@ public String getRoomProperty(@RequestBody Integer roomId) {
         return null;
     }
 
-    @PostMapping("makenewbooking")
+    @PostMapping(value ="makenewbooking", produces = "application/json; charset=UTF-8")
+    @ResponseBody
```

Comment:
Yes. This is a RestController, so there is implicit ResponseBody. You needn't explicitly write it.